modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -1,87 +1,129 @@
+a|
+. In the *Transmission Flow* section, click *View Payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the MDN message sent to the partner
+a|
+. In the *Transmission Flow* section, click *View MDN*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the CONTRL message sent to the partner
+a|
+. In the *Transmission Flow* section, click *View CONTRL*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the payload sent to the backend
+a|
+. In the message flow diagram, expand the *Source* section.
+. Click *View Payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+|View the message flow diagram for the transmitted message
+|Click the link in the *Processed message through message flows* field.
+{sp} +
+{sp}+
+From there, you can expand the message flow sections to view additional information, including information about the receiving (Receive from Partners) endpoint, DataWeave map, target message type, and target (Target at Host) endpoint in the message flow.
+|===
+
+== View Details for an Outbound Transmission
+
+The summary information for an outbound transmission includes the following:
+
+* Transmission status (a green checkmark for success or a red exclamation point for failure)
+* Transmission ID
+* Date and time when the file was sent
+* Control number associated with the transmitted file
+* Message type, such as `mythical-po-ack-json: ORD-9892330`
+* Name of the sending (Send to Partners) endpoint
+* Name of the transmitted file
+
+Take any of the following actions to view and download additional information about the outbound transmission:
+
+|===
+|Action |Steps
+
+|View information about the sending (Send to Partner) endpoint | In the *Summary* section, click the link in the *Endpoint* field.
+|View and download the transmission metadata
+a|
+. In the *Summary* section, click the link in the *File* field.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the payload received from the backend
+a|
+. In the message flow diagram, expand the *Source* section.
+. Click *View Payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the payload sent to the partner
+a|
+. In the message flow diagram, expand the *Sent to* section.
+. In the *Sent transmission* field, click *View payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+| View and download the MDN received from the partner
+a|
+. In the message flow diagram, expand the *Sent to* section.
+. In the *Received MDN* field, click *View payload*.
+. Specify a wrap character, if needed.
+. Click *Download*.
+|View the CONTRL message received from the partner, if it exists
+a|
+. In the message flow diagram, expand the *Sent to* section.
+. In the ??? field *Click *View payload*. (*Vijay, what is the name of this field? I only see the Waiting for CONTRL status here so I can't tell.*)
```

Comment:
This is same as the experience in X12. 
In the *Sent transmission* field, click *View payload*.